hat/assets/js/apps/Iaso/domains/pages/index.js

```diff
@@ -190,14 +99,18 @@ const Pages = () => {
             />
             <TopBar title={formatMessage(MESSAGES.pages)} />
             <Box className={classes.containerFullHeightNoTabPadded}>
-                <PageActions>
-                    <PageAction
-                        icon={AddIcon}
-                        onClick={handleClickCreateButton}
-                    >
-                        {formatMessage(MESSAGES.create)}
-                    </PageAction>
-                </PageActions>
+                <Filters params={params} />
+                <Box mt={4}>
+                    <PageActions>
+                        <PageAction
+                            icon={AddIcon}
+                            onClick={handleClickCreateButton}
+                        >
+                            {formatMessage(MESSAGES.create)}
+                        </PageAction>
+                    </PageActions>
+                </Box>
+
```

Comment:
If you have time for it, you could get rid of `PageActions` and `PageAction`. These components are legacy (wink @beygorghor ) and are obnly still used here.
You could just just replace them with the underlying components, which are just `Grid` and `Button`, don't add that many lines of code, and would make this component's style consistent with the rest of iaso.

Nit blocking but much appreciated if done 😄